src/lib/auth-helper.js

```diff
@@ -47,8 +47,8 @@ async function getAccessToken ({ useCachedToken = false } = {}) {
 
   let accessToken = null
   if (useCachedToken) {
-    const contextConfig = await context.get(contextName)
-    accessToken = contextConfig?.access_token?.token
+    const { data } = await context.get(contextName)
```

Comment:
we lost some clarity here for shortness?

Reply:
actually no, it was a bug. as you can see data was not dereferenced